src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+import mate.academy.spring.model.User;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketsId;
+    private User user;
```

Comment:
don't return object of real model, this field can be removed completely and replaced with userId